internal/windows/displays_windows.go

```diff
@@ -19,19 +19,36 @@ func (d *driver) Displays() []*webapp.Display {
 			var dpiX, dpiY uint32
 			if err = GetDpiForMonitor(monitor, MDT_EFFECTIVE_DPI, &dpiX, &dpiY); err != nil {
 				jot.Error(err)
```

Comment:
No need to log here since the code below handles the error.

Reply:
Got it